worlds/osrs/__init__.py

```diff
@@ -524,7 +524,10 @@ def create_region(self, name: str) -> "Region":
         return region
 
     def create_item(self, item_name: str) -> "Item":
-        item = [item for item in item_rows if item.name == item_name][0]
+        items = [item for item in item_rows if item.name == item_name]
+        if len(items) == 0:
```

Comment:
Do you expect to actually hit this in AP releases? You could use an assert instead

Reply:
I had asserts in other spots before but was suggested to raise an exception instead. I'm not entirely sure the difference, is this a situation where assert would be better?

Reply:
An assert will not happen in released versions of AP. Only in source/dev versions

Reply:
Fair enough. This wouldn't be a runtime exception so it's been replaced with an assert